ui/apps/platform/src/Containers/Policies/Table/PoliciesTablePage.tsx

```diff
@@ -178,6 +186,37 @@ function PoliciesTablePage({
 
     return (
         <>
+            <PageTitle title="Policy management" />
+            <PageSection variant="light">
+                <Title headingLevel="h1">Policy Management</Title>
+            </PageSection>
+            <PageSection variant="light" className="pf-u-px-sm pf-u-py-0">
+                <TabNav
+                    currentTabTitle="Policies"
+                    tabLinks={[{ title: 'Policies', link: policiesBasePath }]}
+                />
+            </PageSection>
+            <Divider component="div" />
+            <PageSection variant="light" className="pf-u-py-md">
+                <Split>
+                    <SplitItem isFilled>
```

Comment:
Rather than the Split-SplitItem pattern we used in Access section, we have more recently been using Toolbar-ToolbarItem for this.

Example, 
`ui/apps/platform/src/Containers/VulnMgmt/Reports/VulnMgmtReportTablePage.tsx`

Mark may know other instances